app/controllers/posts_controller.rb

```diff
@@ -4,11 +4,22 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts.order(created_at: :desc)
+
+    respond_to do |format|
+      format.html
+      format.xml { render xml: @posts }
+      format.json { render json: @posts }
+    end
   end
 
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
+    respond_to do |format|
+      format.html
+      format.xml { render xml: @post.comments }
+      format.json { render json: @post.comments }
+    end
   end
```

Comment:
- [ ] Great work on implementing the API endpoints but I guess misunderstood the requirements :fearful: The requirement was to add more API endpoints to use your application in a different way means mobile applications or other apps can access your data but it means that you original app should work the same :thinking: But here you have changed the original app :fearful: It would be nice if you add your API endpoints in different files by this your original app will not be affected :+1: 

Now if you will visit urls that are triggering these methods then they won't give you the JSON objects but they will take you to your app views or maybe your app gets confused :fearful:


:green_circle: You can visit this [pull request](https://github.com/dennis-every/blog-app/pull/11/files) to get an idea what i am talking about :+1: